### PR TITLE
docs: Product vocabulary list

### PR DESCRIPTION
**Pull request by agglasaur**
copy and paste from branch i oopsed on

### Motivations
So we can be consistent with a shared language in Jobba.

### Note
The not-ideal styling of this list is a known issue and will be resolved in the future.

Resurrection from this failure: https://github.com/GetJobber/atlantis/pull/442

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Comment by agglasaur** [comment]
@chris-at-jobber 🤑 🤑 👋  Review pls

File: docs/guides/product-vocabulary.mdx [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+
+Avoid **please** for the most part, particularly in error messages to avoid
+overuse and loss of meaning.
+
+| **✅ Do**        | **❌ Don't**            |
+| ---------------- | ----------------------- |
+| Enter your email | Please enter your email |
+
+Use **please** sparingly when we're requesting the user to take additional
+effort to resolve an issue, outside of standard errors.
```
Comment (chris-at-jobber) [inline docs/guides/product-vocabulary.mdx:481]: image
:sob_lol:

File: docs/guides/product-vocabulary.mdx [hunk 2]
```diff
@@ -0,0 +1,532 @@
...
+
+| **✅ Do**                              | **❌ Don't**                                      |
+| -------------------------------------- | ------------------------------------------------- |
+| Stored cards will need to be re\-added | Any vaulted cards will need to be re\-established |
+
+<br />
+
+## Concepts and Items
+
+In general, none of these should be capitalized unless explicitely stated or
```
Comment (chris-at-jobber) [inline docs/guides/product-vocabulary.mdx:314]: typo: `explicitely`

File: docs/guides/product-vocabulary.mdx [hunk 3]
```diff
@@ -0,0 +1,532 @@
...
+| **✅ Do**                                          | **❌ Don't**                                |
+| -------------------------------------------------- | ------------------------------------------- |
+| No team members are currently assigned to this job | No users are currently assigned to this job |
+
+Never use **fieldworker** as this does not cover all company roles
+appropriately.
+
+<br />
+
+### we
```
Comment (chris-at-jobber) [inline docs/guides/product-vocabulary.mdx:86]: This kind of ties into something lower down about "Contact **our** Success team", is that still OK in this context? We talk about `we` and `you / you're` but `your / our` might be a good pairing to add in here (that was a really confusing sentence to write)

**Comment by chris-at-jobber** [comment]
@agglasaur one lil' typo and one pair of pronouns that I think would help!

File: docs/guides/product-vocabulary.mdx (hunk 1 above)
Comment (agglasaur) [reply docs/guides/product-vocabulary.mdx:481]: _sparingly_

File: docs/guides/product-vocabulary.mdx (hunk 3 above)
Comment (agglasaur) [reply docs/guides/product-vocabulary.mdx]: I combined `we / our` as it seemed like a natural pairing also but i did add a note to the client hub portion for the pronoun usage (thats the one we seem to go back and forth the most on in Jobber online from my gander)

**Comment by agglasaur** [comment]
@chris-at-jobber Thank you for the thoroughness, ready for another look!

**Review by chris-at-jobber — APPROVED** [review]
Looks great to me! 🚢

**Pull request merged**